CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ All notable changes to [camunda-bpmn-js](https://github.com/camunda/camunda-bpmn
 
 ___Note:__ Yet to be released changes appear here._
 
+## 1.1.2
+
+* `DEPS`: update to `bpmn-js-properties-panel@1.15.1`
+
+### Key changes in Properties Panel
+
+* `FIX`: don't stop propagation of events ([bpmn-js-properties-panel#850](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/850))
```

Comment:
```suggestion
* `FIX`: correct clear and set of errors ([bpmn-js-properties-panel#850](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/850))
```

Reply:
Makes sense to writte it from the user-perspective here 👍